research/combine_aq_airports_data.py

```diff
@@ -0,0 +1,33 @@
+"""
+Program to inject air quality data into the airports list from FlightRadar24
+"""
+
+import json
+import csv
+
+# assign airports json to local object
+with open('../data/airports.json', 'r') as f:
+    airports_orig = json.load(f)
+
+# assign air quality data to object
+air_qualities = []
+with open('../data/airports_aq_large.csv', 'r') as f:
+    reader = csv.reader(f)
+    for row in reader:
+        air_qualities.append(row[0].split("\t"))
```

Comment:
Why do you need to split the line? I think CSV should be able to do this for you?